hopsworks-IT/src/test/ruby/spec/featurestore_spec.rb

```diff
@@ -904,6 +965,57 @@
           expect(parsed_json2["dataFormat"] == "petastorm").to be true
         end
 
+        it "should be able to get a list of training dataset versions based on the name" do
+          project = get_project
+          featurestore_id = get_featurestore_id(project.id)
+          connector = get_hopsfs_training_datasets_connector(@project[:projectname])
+          json_result, training_dataset_name = create_hopsfs_training_dataset(project.id, featurestore_id, connector)
+          expect_status(201)
+
+          json_result, training_dataset_name = create_hopsfs_training_dataset(project.id, featurestore_id, connector, name: training_dataset_name, version: 2)
+          expect_status(201)
+
+          # Get the list  
+          get_training_datasets_endpoint = "#{ENV['HOPSWORKS_API']}/project/#{project.id}/featurestores/#{featurestore_id}/trainingdatasets/name/#{training_dataset_name}"
+          get get_training_datasets_endpoint 
+          parsed_json = JSON.parse(response.body)
+          expect_status(200)
+          expect(parsed_json.size).to eq 2
+        end
+
+        it "should be able to get a training dataset based on name and version" do
+          project = get_project
+          featurestore_id = get_featurestore_id(project.id)
+          connector = get_hopsfs_training_datasets_connector(@project[:projectname])
+          json_result, training_dataset_name = create_hopsfs_training_dataset(project.id, featurestore_id, connector)
+          expect_status(201)
+
+          json_result, training_dataset_name = create_hopsfs_training_dataset(project.id, featurestore_id, connector, name: training_dataset_name, version: 2)
+          expect_status(201)
+
+          # Get the list  
+          get_training_datasets_endpoint = "#{ENV['HOPSWORKS_API']}/project/#{project.id}/featurestores/#{featurestore_id}/name/trainingdatasets/#{training_dataset_name}/version/1"
```

Comment:
```suggestion
          get_training_datasets_endpoint = "#{ENV['HOPSWORKS_API']}/project/#{project.id}/featurestores/#{featurestore_id}/trainingdatasets/name/#{training_dataset_name}/version/1"
```
Shouldn't it be `.../trainingdatasets/name/...`?